### ISSUE TITLE
top 2 rows not showing up on stats window

### ISSUE THREAD

**Issue by bermanskid** [ShootMe/FallGuysStats#198]
Hi there -

With this new update, I'm getting two long red X boxes for the two rows up where settings and show type/time played are supposed to be. I can still click the box in various areas to access the settings but can't see them.

**Comment by qutrits** [comment]
Uploading a screenshot of the screen where the problem occurs is also helpful in troubleshooting.

**Comment by bermanskid** [comment]
![　  Fall Guys Stats v1 139 3_25_2023 1_24_26 AM](https://user-images.githubusercontent.com/117063112/227706237-8b82944d-f958-4711-bb0d-f639893b21d4.png)

**Comment by qutrits** [comment]
@bermanskid First of all, is that what happens when you download the latest version and run it?

And what OS are you currently using?

**Comment by bermanskid** [comment]
yes my fallguysstats.exe updates automatically but I also downloaded it to see if that would work. 

I'm using windows 11 v.22H2

Also, I get this screen every time I open the program. The program runs fine when I'm playing the game, so I don't really think of it as an issue but just wanted to bring it to attention.

![Screenshot 2023-03-25 164718](https://user-images.githubusercontent.com/117063112/227747772-45a9c389-c73b-45cf-b968-858a54b2dc69.png)

**Comment by qutrits** [comment]
I'm using the same version of the os, but I can't reproduce the issues. Could you click Details to send me the full details?

And could you try something like below?
step 1. Download the latest version of FallGuysStats.zip from the main screen
step 2. Extract the downloaded zip file
step 3. Put the data.db file in the existing FallGuysStats folder into the unzipped folder and run the FallGuysStats.exe file

**Comment by bermanskid** [comment]
Tried the instructions... same issue still pops up unfortunately.

**Comment by iku55** [comment]
> yes my fallguysstats.exe updates automatically but I also downloaded it to see if that would work.
> 
> I'm using windows 11 v.22H2
> 
> Also, I get this screen every time I open the program. The program runs fine when I'm playing the game, so I don't really think of it as an issue but just wanted to bring it to attention.
> 
> ![Screenshot 2023-03-25 164718](https://user-images.githubusercontent.com/117063112/227747772-45a9c389-c73b-45cf-b968-858a54b2dc69.png)

Could you tell me details of this error?
Also, does this issue still occur if you move data.db to another location?

**Comment by qutrits** [comment]
See the end of this message for details on invoking

just-in-time (JIT) debugging instead of this dialog box.

 

************** Exception Text **************

System.NullReferenceException: Object reference not set to an instance of an object.

   at MetroFramework.MetroFonts.get_Title()

   at MetroFramework.Forms.MetroForm.OnPaint(PaintEventArgs e)

   at System.Windows.Forms.Control.PaintTransparentBackground(PaintEventArgs e, Rectangle rectangle, Region transparentRegion)

   at System.Windows.Forms.Control.PaintBackground(PaintEventArgs e, Rectangle rectangle, Color backColor, Point scrollOffset)

   at System.Windows.Forms.Control.PaintBackground(PaintEventArgs e, Rectangle rectangle)

   at System.Windows.Forms.Control.OnPaintBackground(PaintEventArgs pevent)

   at System.Windows.Forms.ScrollableControl.OnPaintBackground(PaintEventArgs e)

   at System.Windows.Forms.ToolStrip.OnPaintBackground(PaintEventArgs e)

   at System.Windows.Forms.Control.PaintWithErrorHandling(PaintEventArgs e, Int16 layer)

   at System.Windows.Forms.Control.WmPaint(Message& m)

   at System.Windows.Forms.Control.WndProc(Message& m)

   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)

   at System.Windows.Forms.ToolStrip.WndProc(Message& m)

   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)

   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)

   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

 

 

************** Loaded Assemblies **************

mscorlib

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B

    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll

----------------------------------------

FallGuysStats

    Assembly Version: 1.140.0.0

    Win32 Version: 1.140.0.0

    CodeBase: file:///C:/Users/jlmcs/OneDrive/Desktop/Games/Fall%20Guys%20Stats/FallGuysStats.exe

----------------------------------------

System

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll

----------------------------------------

MetroFramework

    Assembly Version: 1.4.0.0

    Win32 Version: 1.140.0.0

    CodeBase: file:///C:/Users/jlmcs/OneDrive/Desktop/Games/Fall%20Guys%20Stats/FallGuysStats.exe

----------------------------------------

System.Windows.Forms

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9075.0 built by: NET481REL1LAST_C

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll

----------------------------------------

System.Drawing

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9032.0 built by: NET481REL1

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll

----------------------------------------

LiteDB

    Assembly Version: 5.0.9.0

    Win32 Version: 5.0.9

    CodeBase: file:///C:/Users/jlmcs/OneDrive/Desktop/Games/Fall%20Guys%20Stats/LiteDB.DLL

----------------------------------------

System.Core

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll

----------------------------------------

System.Configuration

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9032.0 built by: NET481REL1

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll

----------------------------------------

System.Xml

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9032.0 built by: NET481REL1

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll

----------------------------------------

Anonymously Hosted DynamicMethods Assembly

    Assembly Version: 0.0.0.0

    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_32/mscorlib/v4.0_4.0.0.0__b77a5c561934e089/mscorlib.dll

----------------------------------------

System.IO.Compression

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9032.0 built by: NET481REL1

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.dll

----------------------------------------

System.IO.Compression.FileSystem

    Assembly Version: 4.0.0.0

    Win32 Version: 4.8.9032.0

    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression.FileSystem/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.FileSystem.dll

----------------------------------------

 

************** JIT Debugging **************

To enable just-in-time (JIT) debugging, the .config file for this

application or computer (machine.config) must have the

jitDebugging value set in the system.windows.forms section.

The application must also be compiled with debugging

enabled.

 

For example:

 

<configuration>

    <system.windows.forms jitDebugging="true" />

</configuration>

 

When JIT debugging is enabled, any unhandled exception

will be sent to the JIT debugger registered on the computer

rather than be handled by this dialog box.

**Comment by qutrits** [comment]
See the end of this message for details on invoking
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at MetroFramework.MetroFonts.get_Title()
   at MetroFramework.Forms.MetroForm.OnPaint(PaintEventArgs e)
   at System.Windows.Forms.Control.PaintTransparentBackground(PaintEventArgs e, Rectangle rectangle, Region transparentRegion)
   at System.Windows.Forms.Control.PaintBackground(PaintEventArgs e, Rectangle rectangle, Color backColor, Point scrollOffset)
   at System.Windows.Forms.Control.PaintBackground(PaintEventArgs e, Rectangle rectangle)
   at System.Windows.Forms.Control.OnPaintBackground(PaintEventArgs pevent)
   at System.Windows.Forms.ScrollableControl.OnPaintBackground(PaintEventArgs e)
   at System.Windows.Forms.ToolStrip.OnPaintBackground(PaintEventArgs e)
   at System.Windows.Forms.Control.PaintWithErrorHandling(PaintEventArgs e, Int16 layer)
   at System.Windows.Forms.Control.WmPaint(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
FallGuysStats
    Assembly Version: 1.140.0.0
    Win32 Version: 1.140.0.0
    CodeBase: file:///C:/Users/jlmcs/OneDrive/Desktop/Games/Fall%20Guys%20Stats/FallGuysStats.exe
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
MetroFramework
    Assembly Version: 1.4.0.0
    Win32 Version: 1.140.0.0
    CodeBase: file:///C:/Users/jlmcs/OneDrive/Desktop/Games/Fall%20Guys%20Stats/FallGuysStats.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9075.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
LiteDB
    Assembly Version: 5.0.9.0
    Win32 Version: 5.0.9
    CodeBase: file:///C:/Users/jlmcs/OneDrive/Desktop/Games/Fall%20Guys%20Stats/LiteDB.DLL
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 4.8.9139.0 built by: NET481REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_32/mscorlib/v4.0_4.0.0.0__b77a5c561934e089/mscorlib.dll
----------------------------------------
System.IO.Compression
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.dll
----------------------------------------
System.IO.Compression.FileSystem
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression.FileSystem/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.FileSystem.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example: 

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

**Comment by bermanskid** [comment]
Yes the error still occurs even when I move the data.db file, at least as of my last posting. Not sure what the last post(s) refer to, so if further explanation is possible in layman’s terms I’d appreciate it heheh. Thanks!On Mar 30, 2023, at 5:42 AM, iku55 ***@***.***> wrote:﻿

yes my fallguysstats.exe updates automatically but I also downloaded it to see if that would work.
I'm using windows 11 v.22H2
Also, I get this screen every time I open the program. The program runs fine when I'm playing the game, so I don't really think of it as an issue but just wanted to bring it to attention.


Could you tell me details of this error?
Also, does this issue still occur if you move data.db to another location?

**Comment by borg0borg** [comment]
I actually have the same issue

**Comment by qutrits** [comment]
Does the same problem appear in version 1.143?

**Comment by borg0borg** [comment]
Yes. Same issue.
![image](https://user-images.githubusercontent.com/48465722/229368021-38055cb8-6069-4439-b961-69d0b13b9c24.png)

**Comment by qutrits** [comment]
@borg0borg Could you try installing .NET Framework 4.8 from the link below and running it after rebooting?

Download .NET Framework 4.8
[web installer](https://dotnet.microsoft.com/ko-kr/download/dotnet-framework/thank-you/net48-web-installer)
[offline installer](https://dotnet.microsoft.com/ko-kr/download/dotnet-framework/thank-you/net48-offline-installer)

**Comment by borg0borg** [comment]
I can't install it, unfortunately
![image](https://user-images.githubusercontent.com/48465722/229830601-24734946-b205-41f8-abd6-fccd29699730.png)

**Comment by qutrits** [comment]
While looking for a solution, I found the case below.

[red-cross-problem-on-menustrip-and-toolstrip](https://stackoverflow.com/questions/1000052/red-cross-problem-on-menustrip-and-toolstrip)

[how-to-manage-virtual-memory-on-windows-11](https://www.windowscentral.com/software-apps/windows-11/how-to-manage-virtual-memory-on-windows-11)

**Comment by borg0borg** [comment]
I don't think it's caused by the virtual memory as I have 64GB of page file size set up on my computer. I also have 32GB of RAM installed.

**Comment by iku55** [comment]
Does this work? [FallGuysStats.zip](https://github.com/ShootMe/FallGuysStats/files/11171429/FallGuysStats.zip)
Also, can you check which version of the software causes this problem?
You can download the zip of each version from [this link](https://github.com/ShootMe/FallGuysStats/commits/master/FallGuyStats.zip).
(View at this point in the history -> View raw)
![image](https://user-images.githubusercontent.com/51874867/230405185-105423af-1fd6-412b-8115-eeb7f100d23c.png)

**Comment by bermanskid** [comment]
I've tried 1.139, 1.141 and 1.143 and they all have the same issue.

Also tried adjusting virtual memory but that didn't work.

**Comment by iku55** [comment]
Thank you. I am not sure why this problem is occurring, but it appears to be occurring in this area.
![screenshot 1](https://user-images.githubusercontent.com/51874867/230578295-d2559280-01a6-4e53-b681-b0394f5fa390.png)
https://github.com/ShootMe/FallGuysStats/blob/6030f368ba5998ed9e5300a7da2546e8ed06e053/Views/Stats.cs#L451-L453
This code displays an image indicating that the profile is linked.
![screenshot 2](https://user-images.githubusercontent.com/51874867/230580012-82b73eb7-5695-487f-8f89-56c7a9486c18.png)

**Comment by borg0borg** [comment]
> Does this work? [FallGuysStats.zip](https://github.com/ShootMe/FallGuysStats/files/11171429/FallGuysStats.zip) Also, can you check which version of the software causes this problem? You can download the zip of each version from [this link](https://github.com/ShootMe/FallGuysStats/commits/master/FallGuyStats.zip). (View at this point in the history -> View raw) ![image](https://user-images.githubusercontent.com/51874867/230405185-105423af-1fd6-412b-8115-eeb7f100d23c.png)

This version works for me. Can I continue using it?
![image](https://user-images.githubusercontent.com/48465722/230661146-ddd584ab-8774-40b6-8a20-bc2cd641d059.png)

I also tried the other versions. Same issue from 1.136 onwards except for 1.137.
![image](https://user-images.githubusercontent.com/48465722/230663429-9f4eb11d-3d78-41d7-911f-3e8efdabaa78.png)

**Comment by borg0borg** [comment]
The app automatically updated to v1.146 and the issue's back. :(

**Comment by iku55** [comment]
Could you please upload the error.log created by running this?
[FallGuysStats.zip](https://github.com/ShootMe/FallGuysStats/files/11185073/FallGuysStats.zip)

**Comment by qutrits** [comment]
@borg0borg It could be a cache issue, could you try the following programs?

[Microsoft PC Manager](https://pcmanager-en.microsoft.com/)

**Comment by iku55** [comment]
> > Could you please upload the error.log created by running this? [FallGuysStats.zip](https://github.com/ShootMe/FallGuysStats/files/11185145/FallGuysStats.zip)
> 
> I think I found something. I'm not sure if it has something to do with my antivirus because when I tried running the app, my antivirus blocked it. So I added it to the exception list, and I was able to run it with no errors.

Was error.log created? Also, what is your antivirus?

**Comment by borg0borg** [comment]
No. There was no error.log created. I have McAfee LiveSafe. Encountered the error again after the app auto-updated to v1.148. McAfee removed the exception due to the file update, and the error reappeared. I added it back to the exception list and was able to solve the problem.

**Comment by bermanskid** [comment]
So this problem reappeared recently w/ v.1.294. I've had the file added to the exception list for McAfee but now the top 3 rows are X'd out like the two rows were before. Updates unfortunately have not fixed the issue either unfortunately.

**Comment by borg0borg** [comment]
> So this problem reappeared recently w/ v.1.294. I've had the file added to the exception list for McAfee but now the top 3 rows are X'd out like the two rows were before. Updates unfortunately have not fixed the issue either unfortunately.

yup. same. 

here's the details of the error:
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at MetroFramework.MetroFonts.get_Title()
   at MetroFramework.Forms.MetroForm.OnPaint(PaintEventArgs e)
   at System.Windows.Forms.Control.PaintTransparentBackground(PaintEventArgs e, Rectangle rectangle, Region transparentRegion)
   at System.Windows.Forms.Control.PaintBackground(PaintEventArgs e, Rectangle rectangle, Color backColor, Point scrollOffset)
   at System.Windows.Forms.Control.PaintBackground(PaintEventArgs e, Rectangle rectangle)
   at System.Windows.Forms.Control.OnPaintBackground(PaintEventArgs pevent)
   at System.Windows.Forms.ScrollableControl.OnPaintBackground(PaintEventArgs e)
   at System.Windows.Forms.ToolStrip.OnPaintBackground(PaintEventArgs e)
   at System.Windows.Forms.Control.PaintWithErrorHandling(PaintEventArgs e, Int16 layer)
   at System.Windows.Forms.Control.WmPaint(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.MenuStrip.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9181.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
FallGuysStats
    Assembly Version: 1.297.0.0
    Win32 Version: 1.297.0.0
    CodeBase: file:///E:/%C2%A3%C3%A3%C2%AE%C3%A3%C3%9F%C3%B5%C2%AEg/Facebook%20Gaming/Installers/FallGuysStats/FallGuysStats.exe
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9172.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
MetroFramework
    Assembly Version: 1.4.0.0
    Win32 Version: 1.297.0.0
    CodeBase: file:///E:/%C2%A3%C3%A3%C2%AE%C3%A3%C3%9F%C3%B5%C2%AEg/Facebook%20Gaming/Installers/FallGuysStats/FallGuysStats.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9181.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9181.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
LiteDB
    Assembly Version: 5.0.17.0
    Win32 Version: 1.297.0.0
    CodeBase: file:///E:/%C2%A3%C3%A3%C2%AE%C3%A3%C3%9F%C3%B5%C2%AEg/Facebook%20Gaming/Installers/FallGuysStats/FallGuysStats.exe
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 4.8.9181.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/mscorlib/v4.0_4.0.0.0__b77a5c561934e089/mscorlib.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9191.0 built by: NET481REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
System.Numerics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0 built by: NET481REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.Text.Json
    Assembly Version: 7.0.0.3
    Win32 Version: 1.297.0.0
    CodeBase: file:///E:/%C2%A3%C3%A3%C2%AE%C3%A3%C3%9F%C3%B5%C2%AEg/Facebook%20Gaming/Installers/FallGuysStats/FallGuysStats.exe
----------------------------------------
System.Memory
    Assembly Version: 4.0.1.2
    Win32 Version: 1.297.0.0
    CodeBase: file:///E:/%C2%A3%C3%A3%C2%AE%C3%A3%C3%9F%C3%B5%C2%AEg/Facebook%20Gaming/Installers/FallGuysStats/FallGuysStats.exe
----------------------------------------
System.ValueTuple
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.9032.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.ValueTuple/v4.0_4.0.0.0__cc7b13ffcd2ddd51/System.ValueTuple.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
